.github/workflows/linux.yml

```diff
@@ -16,474 +16,445 @@ concurrency:
-          # github.ref_name is 'ref/PR_#' in case of the PR, and 'branch_name' when executed on push
-          save-always: ${{ github.ref_name == 'master' && 'true' || 'false'  }}
-          path: ${{ env.CCACHE_DIR }}
-          key: ${{ runner.os }}-${{ runner.arch }}-ccache-ov-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-${{ runner.arch }}-ccache-ov
-
-      - name: CMake configure - OpenVINO
-        run: |
-          cmake \
-            -G "${{ env.CMAKE_GENERATOR }}" \
-            -DENABLE_CPPLINT=OFF \
-            -DENABLE_NCC_STYLE=OFF \
-            -DENABLE_TESTS=OFF \
-            -DENABLE_STRICT_DEPENDENCIES=OFF \
-            -DENABLE_SYSTEM_OPENCL=ON \
-            -DCMAKE_VERBOSE_MAKEFILE=ON \
-            -DCPACK_GENERATOR=TGZ \
-            -DENABLE_JS=OFF \
-            -DENABLE_SAMPLES=ON \
-            -DENABLE_OV_ONNX_FRONTEND=OFF \
-            -DENABLE_OV_PADDLE_FRONTEND=OFF \
-            -DENABLE_OV_PYTORCH_FRONTEND=ON \
-            -DENABLE_OV_TF_FRONTEND=ON \
-            -DENABLE_OV_TF_LITE_FRONTEND=OFF \
-            -DENABLE_INTEL_GPU=OFF \
-            -DENABLE_INTEL_NPU=ON \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-            -DENABLE_PYTHON=ON \
-            -DENABLE_WHEEL=ON \
-            -S ${OPENVINO_REPO} \
-            -B ${BUILD_DIR}
-
-      - name: Clean ccache stats
-        run: ccache --zero-stats --show-config
-
-      - name: Cmake build - OpenVINO
-        run: cmake --build ${BUILD_DIR} --parallel --config ${{ env.CMAKE_BUILD_TYPE }}
-
-      - name: Show ccache stats
-        run: ccache --show-stats
-
-      - name: Cmake install - OpenVINO
-        run: |
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}/openvino_package -P ${BUILD_DIR}/cmake_install.cmake
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}/openvino_package -DCOMPONENT=python_wheels -P ${BUILD_DIR}/cmake_install.cmake
-
-      - name: Pack Artifacts
-        run: |
-          pushd ${INSTALL_DIR}
-            tar -czvf ${BUILD_DIR}/openvino_package.tar.gz *
-          popd
-
-      #
-      # Upload build artifacts and logs
-      #
-
-      - name: Upload openvino package
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: openvino_package
-          path: ${{ env.BUILD_DIR }}/openvino_package.tar.gz
-          if-no-files-found: 'error'
-
-  genai_python_lib:
-    name: OpenVINO genai extension (cmake + wheel)
-    needs: [ openvino_download, openvino_build ]
-    if: |
-      always() &&
-      (needs.openvino_download.outputs.status == 'success' || needs.openvino_build.result == 'success')
+    - uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
+      id: openvino_download
+      with:
+        platform: 'ubuntu22'
+        commit_packages_to_provide: 'wheels'
+        revision: 'latest_available_commit'
```

Comment:
currently we use hardocoded commit 0080d90974ca84f9a6d359da3388a2a18a93b753, because with newer ones OpenVINO GenAI tests are broken

Reply:
updated